docs/environment-variables.md

```diff
@@ -117,10 +117,12 @@ those.
   this variable is set to any value, `graph-node` will still accept GraphQL
   subscriptions, but they won't receive any updates.
 - `ENABLE_GRAPHQL_VALIDATIONS`: enables GraphQL validations, based on the GraphQL specification.
-  This will validate and ensure every query executes follows the execution rules.
+  This will validate and ensure every query executes follows the execution
+  rules. Default: `false`
 - `SILENT_GRAPHQL_VALIDATIONS`: If `ENABLE_GRAPHQL_VALIDATIONS` is enabled, you are also able to just
   silently print the GraphQL validation errors, without failing the actual query. Note: queries
-  might still fail as part of the later stage validations running, during GraphQL engine execution.
+  might still fail as part of the later stage validations running, during
+  GraphQL engine execution. Default: `false`
```

Comment:
At the moment this one is set to `true` in the hosted service, and the default in code is set to `true` https://github.com/graphprotocol/graph-node/blob/70bee451802054607c8d2beaa7a793829d0f605b/graph/src/env/graphql.rs#L149

Reply:
Woops .. somehow misread the code. Thanks for checking! Just fixed it.